core/photon.py

```diff
@@ -4,7 +4,9 @@
 
 if __name__ == "__main__":
     import sys
-    sys.path.insert(0, @PHOTON_INSTALL_PATH)
+    import os
+    PHOTON_INSTALL_PATH = getattr(sys, '_MEIPASS', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Não sei se essa linha é necessária, além de ser problemática. Pq na hora da instalação ela irá mudar e dependendo de como a pessoa usar o script, pode não funcionar corretamente.

Quais as vantagens você nessa abordagem ao invés da antiga?

Reply:
Sobre esse getattr, coloquei pq já usei o pyinstaller antes e ele tem problemas em achar os arquivos do diretório em q foi instalado. Deve funcionar normal fora do .exe tbm
https://stackoverflow.com/questions/7674790/bundling-data-files-with-pyinstaller-onefile

Ent quando for instalado, o `photon.exe` nn conseguiria achar outros modulos/arquivos sem esse '_MEIPASS'

Sobre dar problemas junto com o `install.py`, se entendi direito, acho q nn aconteceria, pois ele faz o replace de forma q a linha 8 fica assim:
```py
    PHOTON_INSTALL_PATH = r"The/Current/Path/core" # getattr(sys, '_MEIPASS', os.path.dirname(os.path.realpath(__file__)))
```
[comentando o resto](https://github.com/HashLDash/Photon/pull/21/files/5d6455b6b4581d7458a426f42ad0deb08ed5ca8e#diff-a5fa297e66b71ac1a8685caa5c37577774be2c36024ecda7abeb70bb50214d0fR11)

claro, ainda falta testar

Reply:
Sobre a vantagem do photon.py funcionar localmente: quando for transformar em binário não vai ser possível alterar o arquivo, como estamos fazendo agora. Ent penso q essa é a melhor opção

Reply:
Ok, ótimo!